core/src/mast/mod.rs

```diff
@@ -271,4 +267,16 @@ pub enum MastForestError {
     TooManyNodes,
     #[error("node id: {0} is greater than or equal to forest length: {1}")]
     NodeIdOverflow(MastNodeId, usize),
+    #[error(transparent)]
+    Node(#[from] MastNodeError),
```

Comment:
nit: I'd probably rename this to `NodeError`.

But also, I wonder if having `EmptyBasicBlock` error here is good enough (i.e., we won't need to create `MastNodeError`). I can see pros and cons for both approaches - so, don't really have a strong preference.